src/phoenix/session/session.py

```diff
@@ -489,7 +490,9 @@ def active_session() -> Optional[Session]:
     """
     Returns the active session if one exists, otherwise returns None
     """
-    return _session
+    if _session and _session.active:
+        return _session
+    return None
```

Comment:
I think we should probably print something here otherwise they are just gonna get a null pointer exception and it will be hard to understand what the heck happened.

Reply:
The downside is that we also call this internally in px.Client, so if we print a message here, it'll get printed a lot.

Reply:
Unless we make it `px.active_session(suppress_warning=True)`

Reply:
I can do a follow-up ticket on this if needed

Reply:
the status quo of returning a failed or ended session is not useful anyway